### PR TITLE
feat: Update GlobalNotification and InlineNotification to support Heart

### PR DESCRIPTION
**Pull request by dmisdm**
# Objective
Uplift the GlobalNotification and InlineNotification components to support Heart theme.

# Motivation and Context
Part of the overarching rebrand to Heart project 
https://docs.google.com/document/d/1G5iN3zZ0XHc0K5FKIFpa263nGW3l8d-jV2nWf6Gcucs/edit#heading=h.u8txzlzfikjh 

# Screenshots (if appropriate)

### Before
image

image

### After
image
image

# Checklist

- [ ] If this contains visual changes, has it been reviewed by a designer?
- [ ] If this introduces either a new component or a breaking change to an existing component, has it been reviewed by an advocate? (Ask in #prod_design_systems)
- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
- [ ] I have or will communicate these changes to the front end practice

# Additional Considerations
- Does there need to be right-to-left (RTL) options for localization and internationalization?
- Has the test suite been updated?
- Have you done cross-browser testing for our [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)?
- Have you updated any relevant documentation or left useful comments in the code?
- Have you reviewed Culture Amp's Web Accessibility guide [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility)?
- If this contains substantial visual changes, especially far reaching ones, have you unlocked the Chromatic step in the pipeline?
- Have Storybook stories been updated?

**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask in #prod_design_systems to have a design system advocate review the PR to catch any issues.

### CODE REVIEW

File: packages/component-library/components/Notification/_styles.scss [hunk 1]
```diff
@@ -218,7 +243,11 @@ $ca-notification-slide-right: transform $ca-duration-slow ease-out;
   }
 
   %ca-notification---global & {
-    @include kz-typography-paragraph-body;
+    font-weight: $kz-var-typography-paragraph-body-font-weight;
+    font-size: $kz-var-typography-paragraph-body-font-size;
+    line-height: $kz-var-typography-paragraph-body-line-height;
+    letter-spacing: $kz-var-typography-paragraph-body-letter-spacing;
+    top: 0;
```
Comment (dmisdm) [inline packages/component-library/components/Notification/_styles.scss:250]: Is this the right way to "inherit baseline" ?

File: packages/component-library/components/Notification/_styles.scss [hunk 2]
```diff
@@ -59,7 +61,8 @@ $ca-notification-slide-right: transform $ca-duration-slow ease-out;
 
   &%ca-notification---global {
     width: 100%;
-    padding: $ca-grid / 2 15px;
+    display: flex;
+    align-items: center;
```
Comment (jasononeil) [inline packages/component-library/components/Notification/_styles.scss:65]: Is this a nicer way of displaying the same thing, or does this actually change the alignment?

File: packages/component-library/components/Notification/_styles.scss [hunk 3]
```diff
@@ -71,42 +74,45 @@ $ca-notification-slide-right: transform $ca-duration-slow ease-out;
...
-    background: $kz-color-yuzu-100;
-    border-color: $kz-color-yuzu-400;
-    color: $kz-color-wisteria-800;
+    background: $kz-var-color-yuzu-100;
+    border-color: $kz-var-color-yuzu-400;
+    color: $kz-var-color-wisteria-800;
 
     &%ca-notification---global {
-      background-color: $kz-color-yuzu-300;
+      background-color: var($kz-var-id-color-yuzu-200, $kz-var-color-yuzu-300);
```
Comment (jasononeil) [inline packages/component-library/components/Notification/_styles.scss:102]: Add a quick comment to make it easier to find the cleanup

```suggestion
      // Use Yuzu200 for Heart theme, and Yuzu300 for zen (the default fallback)
      background-color: var($kz-var-id-color-yuzu-200, $kz-var-color-yuzu-300);
```

File: packages/component-library/components/Notification/_styles.scss [hunk 4]
```diff
@@ -71,42 +74,45 @@ $ca-notification-slide-right: transform $ca-duration-slow ease-out;
...
-    background: $kz-color-coral-100;
-    border-color: $kz-color-coral-300;
-    color: $kz-color-wisteria-800;
+    background: $kz-var-color-coral-100;
+    border-color: $kz-var-color-coral-300;
+    color: $kz-var-color-wisteria-800;
 
     &%ca-notification---global {
-      background-color: $kz-color-coral-300;
+      background-color: var(
```
Comment (jasononeil) [inline packages/component-library/components/Notification/_styles.scss:112]:
```suggestion
      // Use Coral 200 for Heart theme, and Coral 300 for zen (the default fallback)
      background-color: var(
```

File: packages/component-library/components/Notification/_styles.scss (hunk 1 above)
Comment (jasononeil) [reply packages/component-library/components/Notification/_styles.scss:250]: I would import
`@kaizen/deprecated-component-library-helpers/styles/type.scss` and use `@include inherit-baseline`

**Review by jasononeil — COMMENTED** [review]
Happy with this other than a few small comments

**Comment by ActuallyACat** [comment]
FYI - I've unblocked Chromatic on this PR as this will affect quite a few components: https://github.com/cultureamp/kaizen-design-system/pull/1218/files#diff-1c75702fc5fae25a41477236f91f8fa6d0fa9ec7932dd09b47f88136ab7b7b55L18

**Review by ActuallyACat — CHANGES_REQUESTED** [review]
It looks like the placement of the close icon in the notifications has changed: https://www.chromatic.com/test?appId=5e12c297b9d9020020789ca3&id=6052d773b2c2480021793968

**Comment by dmisdm** [comment]
> It looks like the placement of the close icon in the notifications has changed: https://www.chromatic.com/test?appId=5e12c297b9d9020020789ca3&id=6052d773b2c2480021793968

Ooo good pickup!

**Review by ActuallyACat — APPROVED** [review]
Reran Chromatic and this now LGTM 
https://www.chromatic.com/build?appId=5e12c297b9d9020020789ca3&number=268

**Review by ActuallyACat — APPROVED** [review]

**Pull request merged**